digiwf-libs/digiwf-spring-security/digiwf-spring-security-core/src/main/java/io/muenchendigital/digiwf/spring/security/SpringSecurityProperties.java

```diff
@@ -0,0 +1,25 @@
+package io.muenchendigital.digiwf.spring.security;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "digiwf.security")
+@Data
+public class SpringSecurityProperties {
+  /**
+   * Name of the registration used in main security protection (as resource server).
+   */
+  private String clientRegistration = "keycloak";
+
+  private String[] permittedUrls = {
+      "/error", // allow the error page
```

Comment:
Do we have swagger? Should it be available without auth or should it even be available in prod?

Reply:
you still can override it via properties. For the moment I won't change this.